src/views/CreateRecord/Editor.vue

```diff
@@ -3,6 +3,45 @@
     id="recordEditor"
     fluid
   >
+    <!-- TODO: Loop through the buttons -->
+    <!-- popup to confirm exit from editing -->
+    <v-list>
+      <v-list-item
```

Comment:
is wrapper the dialog boxes inside a list really usefull ?

Reply:
I have no idea which is better. A list seemed like the appropriate choice to me (seems to be used elsewhere in the code). If you want me to change it let me now, otherwise I'll leave it as it is.